README.md

```diff
@@ -90,4 +88,10 @@ For testing we use ExUnit built into Elixir itself
 
 ```sh
 mix test
-```
\ No newline at end of file
+```
+
+## Caveats
+
+Currently this implementation of Crimp does not cover the functionallity that the original project intended, there is an issue with the Notation of Nested Hashes/Maps meaning they get sorted seperately causing the Signature to be incorrect. We are working to fix this and hopefully will update the project soon with that functionality.
```

Comment:
great, I'd be super clear and add an example of what's not working compared to Ruby

Reply:
Good thinking

Reply:
Done!